starter-code/index.js

```diff
@@ -1,11 +1,29 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
-  };
-  
-  module.exports = SortedList;
\ No newline at end of file
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
+  add(item) {
+    this.length = this.items.push(item);
+    this.items.sort(function(a, b) {
+      return a - b;
+    });
+  }
+  get(pos) {
+    return this.items[pos];
+  }
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else return Math.max(...this.items);
+  }
+  min() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else return Math.min(...this.items);
+  }
+  sum() {}
```

Comment:
remember reduce? Try to use it for sum()

and then call back sum() into average() ;)